packages/docs/integrate/extending-integrations.md

```diff
@@ -48,18 +48,66 @@ To register an extension, add it to the array returned from the `extensions` fun
 ```js
 module.exports = {
   integrations: {
-    <TAG NAME>: {
-      location: '@<integration-package>/server',
+    '{INTEGRATION_NAME}': {
+      location: '{SERVER_INTEGRATION}',
       configuration: {},
       extensions: (extensions) => [
         ...extensions,
         {
-          name: 'our-extension',
-          hooks: () => { /* ... */}
+          name: 'extension-name',
+          hooks: () => { /* ... */ }
         }
       ],
       customQueries: {}
     }
   }
 };
 ```
+
+## Example: Adding new API endpoints
+
+You cannot register new API endpoints directly but only through extensions to already registered integrations. Let's follow the examples above to add a new endpoint to the integration.
```

Comment:
To register a new API endpoint, we can register a custom extension and use the `extendApiMethods` property. API endpoints cannot be registered directly.

Let's look at a generic example.